### ISSUE TITLE
General suggestions and discussions on improving structure

### ISSUE THREAD

**Issue by angelosalatino** [napsternxg/awesome-scholarly-data-analysis#6]
Hi,
Since now the taxonomy section is getting large, we might think of dividing it by disciplines, e.g., Computer science, economy, math, and so on. What do you think? In the future this will allow us to easily find out what is missing and what to add.
On the other hand, I realised that you added **SciGraph** as a taxonomy. But actually it is a dataset. I would move it up in "_Publication and Citation_", instead.

Hope this helps

**Comment by napsternxg** [comment]
Hi @angelosalatino I think compared to other sections it is still small, and there is usually going to be 1 or 2 taxonomy for each field, hence the current layout looks good.

**Comment by angelosalatino** [comment]
Ok. I just wanted to give more structure to the list. Maybe we can rearrange the list. Maybe in alphabetical order.

**Comment by napsternxg** [comment]
@angelosalatino added a new issue for your suggestion. Let us use this thread to find possibilities to improve it. However, I want the curation to be minimal effort so that we have higher coverage of content.

**Comment by napsternxg** [comment]
**My core guiding principle for this list is that it should be easy to curate (i.e. can be edited using just the browser), has high coverage of content. I think any improvement or changes to the list should consider that aspect. E.g. it is very easy for me to add new items even from my phone, using the github edit, and this updates the page in realtime.** 

Just wanted to add some personal opinions (I am still open to discussion about these). 

Do you think we can have a json file to curate the list which can be used to automatically generate the content on the webpage? I am not sure how will it affect curating the README automatically. I think that limits the accessibility to edit only for those who know JSON, and also runs the issue of keeping the README out of sync. 

Currently, I could easily add content to README and the website will be auto updated. I am not sure if that can be achieved and the list curation is still user friendly with any automated method.

**Comment by angelosalatino** [comment]
@napsternxg I completely agree with you. Better if we keep it simple (as it is now for instance).
JSON would be cool. However, curating a json dictionary requires us to manipulate it by hand or specific software.
